cognite/neat/_data_model/deployer/_differ_view.py

```diff
@@ -152,7 +184,7 @@ def _diff_edge_property(
             changes.append(
                 ChangedField(
                     field_path=self._get_path(f"{identifier}.edgeSource"),
-                    item_severity=SeverityType.BREAKING,
+                    item_severity=SeverityType.WARNING,
```

Comment:
I am surprised by this. You can change the edge properties, and the API is happy to do so. Should we really allow that without a breaking change? This will break for example pygen.